tools/builder.js

```diff
@@ -150,14 +126,15 @@ const Builder = {
     };
     const outputOptions = {
       file: outPath,
-      format: "commonjs",
+      format: "iife",
       sourcemap: true,
+      strict: true,
```

Comment:
This is redundant as it's true by default per [the documentation](https://rollupjs.org/guide/en/#danger-zone).